docs/zh-cn/dev/user_doc/dev_run.md

```diff
@@ -0,0 +1,113 @@
+#### 搭建 dev 分支开发环境
+
+>    参考：[DolphinScheduler 在 Windows 本地搭建开发环境，源码启动](https://dolphinscheduler.apache.org/zh-cn/blog/DS_run_in_windows.html)
+
+1.   **下载源码**
+
+     GitHub ：https://github.com/apache/dolphinscheduler
+
+     ```shell
+     mkdir dolphinscheduler
+     cd dolphinscheduler
+     git clone git@github.com:apache/dolphinscheduler.git
+     ```
+
+     这里选用 dev 分支。
+
+2.   **Windows 安装 zk**
+
+     1.   下载 zk https://www.apache.org/dyn/closer.lua/zookeeper/zookeeper-3.6.3/apache-zookeeper-3.6.3-bin.tar.gz
+
+     2.   解压 apache-zookeeper-3.6.3-bin.tar.gz
+
+     3.   在 zk 的目录下新建 zkData、zkLog文件夹
+
+     4.   将 conf 目录下的 zoo_sample.cfg 文件，复制一份，重命名为 zoo.cfg，修改其中数据和日志的配置，如：
+
+          ```shell
+          dataDir=D:\\code\\apache-zookeeper-3.6.3-bin\\zkData ## 此处使用绝对路径
+          dataLogDir=D:\\code\\apache-zookeeper-3.6.3-bin\\zkLog
+          ```
+
+     5.   在 bin 中运行 zkServer.cmd，然后运行 zkCli.cmd 查看 zk 运行状态，可以查看 zk 节点信息即代表安装成功。
+
+3.   **搭建后端环境**
```

Comment:
For the database, MySQL is not required, users can choose another(PGSQL). Suggest changes to related words

Reply:
> For the database, MySQL is not required, users can choose another(PGSQL). Suggest changes to related words

I have updated it, please rview~~